src/DrupalDoctrineCache.php

```diff
@@ -26,82 +54,63 @@ public function fetch($id) {
   }
 
   /**
-   * Checks if the Drupal cache contains the specified entry.
-   *
-   * @inheritdoc
+   * {@inheritdoc}
    */
-  public function contains($id) {
-    $entry = $this->fetch($id);
+  protected function doContains($id) {
+    $entry = $this->doFetch($id);
     return !empty($entry);
   }
 
   /**
-   * Saves data to the Drupal cache.
-   *
-   * @inheritdoc
+   * {@inheritdoc}
    */
-  public function save($id, $data, $lifeTime = NULL) {
+  protected function doSave($id, $data, $lifeTime = 0) {
     // Doctrine defines NULL to mean permanent cache.
-    if ($lifeTime == NULL) {
+    if ($lifeTime === NULL) {
```

Comment:
if it is set to 0 by default, shouldn't we consistent and compare against 0 instead?